src/screens/ClipsScreen.tsx

```diff
@@ -254,7 +254,7 @@ export class ClipsScreen extends React.Component<Props, State> {
     })
   }
 
-  _renderClipItem = ({ item, index }) => item && item.episode && item.episode.id ? (
+  _renderClipItem = ({ item, index }) => item?.episode?.id ? (
```

Comment:
I'm a little bit worried about this changes. I'm not sure how a tableview reacts to an undefined item. Before, since it was `item && ...` it would resolve to just false. But this now is a bit different because I don't know what optionals return if they fails. Can you make sure this works ok for the `renderItem` function?

Reply:
You're right. Looks like they return an undefined. When I change

`item?.episode?.id`

to

`item?.episodeasdf?.id`

The ClipsScreen crashes.